content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -79,72 +72,97 @@ npm install
 
-**Checkpoint:** `psql $DATABASE_URL -c "\dt"` should show the list of tables:
+This is part of the development cycle of your app and for all such operations Prisma requires the availability of a [shadow database](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database).
 
-```text
-      List of relations
-Schema |  Name   | Type  |     Owner
---------+---------+-------+----------------
-public | Post    | table | janedoe
-public | Profile | table | janedoe
-public | User    | table | janedoe
-```
+Cloud database providers do not allow the automated creation of a shadow database. To enable a shadow database, you need to create a new database manually in your provider.
+
+After you create a new database, get its connection string.
+
+## 4. Set the SHADOW_DATABASE_URL environment variable locally
+
+Set the `SHADOW_DATABASE_URL` environment variable locally for the shadow database.
 
-Congratulations, you have successfully created the database schema.
+```terminal
+export SHADOW_DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
+```
 
-## 5. Introspect the database
+> **Note:** You need the `SHADOW_DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
 
-Introspect the database with the Prisma CLI:
+Again, replace the uppercase placeholders with your database credentials, like so:
 
 ```terminal
-npx prisma db pull
+postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname
 ```
 
-Prisma will introspect the database defined in the `datasource` block of the Prisma schema and populate the Prisma schema with models corresponding to the database tables.
+Or simply copy and paste the connection string you get from your cloud database provider.
 
-**Checkpoint:** Open `prisma/schema.prisma` and ensure there are three models: `Post`, `Profile`, and `User`.
+## 5. Add the `shadowDatabaseUrl` parameter to `schema.prisma`
 
-## 6. Commit the Prisma schema to the repository
+To enable the use of the shadow database, you must also add the `shadowDatabaseUrl` parameter to the `schema.prisma` file in the `database db` section.
+
+```prisma highlight=4;normal
+datasource db {
+  provider          = "postgresql"
+  url               = env("DATABASE_URL")
+  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
+}
+```
 
-The result of `prisma db pull` is the `schema.prisma` Prisma schema file.
+## 6. Migrate the Prisma schema to the database
 
-Since the Prisma schema is used to generate Prisma Client, commit it to your repository so that Prisma Client can be generated when Netlify builds the project:
+The `prisma/schema.prisma` file contains a pre-defined model for the app. It defines the `Post`, `Profile`, and `User` models which the `prisma migrate` command uses to create the database tables and relationships.
 
 ```terminal
-git commit prisma/schema.prisma -m 'Add Prisma schema'
+npx prisma migrate dev --name init
 ```
```

Comment:
This step `## 6. Migrate the Prisma schema to the database` is pretty short any might require some more explanation what happens there. It also create a folder and files you might want to commit to the fork.

Reply:
If I understand well enough, you are referring to the migrations/ folder and this is no longer an issue now that we say people should use `prisma db push`?